go.mod

```diff
@@ -256,15 +254,15 @@ replace (
 	github.com/go-logfmt/logfmt => github.com/go-logfmt/logfmt v0.4.0
 	github.com/go-logr/logr => github.com/go-logr/logr v0.2.0
 	github.com/go-openapi/analysis => github.com/go-openapi/analysis v0.19.5
-	github.com/go-openapi/errors => github.com/go-openapi/errors v0.19.2
+	github.com/go-openapi/errors => github.com/go-openapi/errors v0.19.5
 	github.com/go-openapi/jsonpointer => github.com/go-openapi/jsonpointer v0.19.3
 	github.com/go-openapi/jsonreference => github.com/go-openapi/jsonreference v0.19.3
 	github.com/go-openapi/loads => github.com/go-openapi/loads v0.19.4
 	github.com/go-openapi/runtime => github.com/go-openapi/runtime v0.19.4
 	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.3
 	github.com/go-openapi/strfmt => github.com/go-openapi/strfmt v0.19.3
 	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.5
-	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.5
+	github.com/go-openapi/validate => github.com/go-openapi/validate v0.19.9-0.20200613183935-c0f6196cdffe
```

Comment:
is there no release yet?

Reply:
There is 0.19.12 already. Maybe we should use that?

Reply:
There is, but that's the problem. The [v0.19.9 release](https://github.com/go-openapi/validate/commits/v0.19.9) that includes my change also includes https://github.com/go-openapi/validate/commit/027c864a983d4f72e957c73b9236e60d7627dd30 which I don't want since it brings a lot of unneeded dep upgrades.

Reply:
@gautierdelorme we do not encourage using SHA when there is a release tag after the SHA. in this case please use one of the release tags after the SHA. if v0.19.9 is not acceptable (say we don’t like the yaml.v2) then we need to work with the upstream project and get it fixed. we can’t pin on this SHA forever right?

Reply:
cc @liggitt @cblecker